host/4/dotnet-isolated-build.yml

```diff
@@ -114,7 +114,7 @@ jobs:
         displayName: $(dotnetVersion)
         continueOnError: false
   - job: Publish
-    dependsOn: Dotnet
+    dependsOn: Dotnet_iso
```

Comment:
this throws --change to dotnet

Reply:
Are you sure? In the previous PR i included the other change but not this change on line 117. I can remove this change but i think part of the reason that this does not run on PRs is because the name

Reply:
Removed them anyway